tarsier/tag_utils.ts

```diff
@@ -424,12 +431,8 @@ function insertTags(
     // into when these empty elements are considered, we should drill
     const childrenToConsider = Array.from(element.childNodes).filter(
       (child) => {
-        if (isNonWhiteSpaceTextNode(child)) {
-          return true;
-        } else if (child.nodeType === Node.TEXT_NODE) {
-          return false;
-        }
-
+        if (isNonWhiteSpaceTextNode(child)) return true;
```

Comment:
we use this throughout the file but let's not one line returns

can lead to errors down the line if we don't have explicit logic for conditions